### DISCUSSION TITLE
Avalonia website should use avalonia to develop

### DISCUSSION THREAD

**Discussion by Xyncgas** [AvaloniaUI/Avalonia#10251] [General]
It would be a show case of confidence if the Avalonia website (https://www.avaloniaui.net/) is made with Avalonia, so people are going to see it's so pretty, and when they look at the source they are not going to see a bunch webpack / jquery / libraries instead they are going to see webassembly or something with the name Avalonia it would encourage developers to try

**Comment by kekekeks** [comment]
Avalonia support for WebAssembly is a technology for creating web _applications_ with the same code that's used for desktop/mobile. It's not supposed to be used for _website_ development.

Websites have a different set of requirements like proper search engine indexing support, way faster load times than .NET webassembly support can provide, etc. So technologies that serve plain HTML directly from the web server (with some hydration in case of websites being powered with SPA frameworks) are preferable.

Web applications usually have quite complex logic and require better maintainability of said logic. Those rarely need indexing and can afford to have some load times.

**Reply by Xyncgas** [reply]
Ok
